context_manager/tests/context_manager_test.py

```diff
@@ -1,6 +1,6 @@
 from unittest import TestCase
 import os
-import cantext_manager as f
+import context_manager as f
```

Comment:
let's import directly without as f

from context_manager import write_wihtou

Reply:
tests for first level is tests/tests_for_level_one.py. context_manager_test.py is old vertion, I will delete it later